### DISCUSSION TITLE
BT-682 Foreign Subsidies Measures not in all view templates for result notices

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#1093] [Q&A] [answered]
Is there a reason why BT-682 is only defined in the view template for form 29 for the winning Tender. Why is it not all result notifes view templates. 
For non winning tenders it is shown in all result notices. But for the winning Tender it is only shown in form 29,

**Comment by pdonohoe** [comment, marked as answer]
Thank you for raising this issue. This is a bug, BT-682 should be displayed for winning Tenders for subtypes 29 to 37. It will be fixed with the next release of the SDK.
